### ISSUE TITLE
Requested record deletes with (RecordDoesNotExist) should not prevent batch submission

### ISSUE THREAD

**Issue by remerle** [vinyldns/vinyldns#1138]
When a user submits a batch change request, and one or more recordsets in the batch change result meet the below criteria, then VinylDNS should allow the batch change to complete successfully.

**Criteria**

- Requested change is a `DeleteRecordSet` change
- The DNS backend does not contain the requested record

In this case, the user's intent is to have the record removed from the DNS backend. If the record doesn't exist, then their request has already been fulfilled.

From a UI perspective, we may want to display an informational message next to the affected records saying something like:

> ℹ️ This record does not exist. No further action is required.

In short, if a user requests the deletion of a record that does not exist, VinylDNS MUST NOT treat this scenario as an error.

**Comment by jhg03a** [comment]
This may also change the behavior of the CLI client as well.
